csclient/params/params.go

```diff
@@ -258,6 +262,30 @@ type WhoAmIResponse struct {
 	Groups []string
 }
 
+// Resource holds the metadata about a Resource defined for a charm.
+type Resource struct {
+	Name        string
+	Type        ResourceType
+	Path        string
+	Description string
+	Revision    int
+	Fingerprint string // the hex encoded hash
+	Size        int64
+}
```

Comment:
Ahh, ok, yeah, I left it out, since OriginStore seemed implied. I can add it, that's fine.